README.md

```diff
@@ -32,4 +32,20 @@ For now, until we solve this problem you can use our project with the test batch
 
 With this you should be able to run the *Cardiac MRI segmentation-checkpoint.ipynb*.
 
-## **Container**
\ No newline at end of file
+## **Container**
+
+### Requirements
+The project can be run through a Docker container. Docker has to be installed on computer, for this to work.
+
+### Start docker
+- After docker is install, simply run **docker_start.ps1** script in a terminal:
+`.\docker_start.ps1`
+- Click the url provided by the docker in the terminal to reach the jupyter server
```

Comment:
- Click the url provided by Docker in the terminal to reach the jupyter server

Reply:
changed to "by the docker container" because it sounds better